apps/external-api-request.yaml

```diff
@@ -0,0 +1,33 @@
+      manifest_cert: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUU2RENDQTlDZ0F3SUJBZ0lDRUFBd0RRWUpLb1pJaHZjTkFRRUxCUUF3Z1lNeEN6QUpCZ05WQkFZVEFsQk0KTVJNd0VRWURWUVFJREFwTllXeHZjRzlzYzJ0aE1ROHdEUVlEVlFRS0RBWkdiMjhnUTI4eEZUQVRCZ05WQkFzTQpERVp2YnlCSmJuUmxjaUJJVVRFU01CQUdBMVVFQXd3SlJtOXZJRWx1ZEdWeU1TTXdJUVlKS29aSWh2Y05BUWtCCkZoUnZabVpwWTJWQWFXNTBaWEl1Wm05dkxtTnZiVEFnRncweU1qQTRNVEF4TWpBMU1qSmFHQTh5TVRJeU1EY3gKTnpFeU1EVXlNbG93Z1k4eEN6QUpCZ05WQkFZVEFrTmFNUkF3RGdZRFZRUUlEQWRDYjJobGJXbGhNUTh3RFFZRApWUVFIREFaUWNtRm5kV1V4RXpBUkJnTlZCQW9NQ2tadmJ5QlNaWEVnUTI4eEV6QVJCZ05WQkFzTUNrWnZieUJTClpYRWdTRkV4RURBT0JnTlZCQU1NQjBadmJ5QlNaWEV4SVRBZkJna3Foa2lHOXcwQkNRRVdFbTltWm1salpVQnkKWlhFdVptOXZMbU52YlRDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTmJFdTNBUgoxVjdjTnUxejFOL0NxUWE3d2ZrUmZ0TUl3RGo4WTIyQ0lpNm04N0lBN3NtRTRWNHhNZUFrSjBHS3RPMldaSW5tCnJOVkRKUG9ldXFjS1YrdGszd0xTdFg4dXQyTmw4dS8rMUNyVHRyV0I1Wi9ONk82NzAxdlVVMzc1RFdWeXJHK2gKWjl6Y21iTFZBZGkvYXpCdVVXYnFjVS9ObnhCeUpPRnlXWEJQTXBCR0tBY09KVGduaVFRMk5ZVVJXaDVmclVvMAp3MXh3Z3ZpWms1OTZrSG8xdDU3a2t4M3E5ZVJ4eWdwS21uS21NZ1I5cFlrOFpTWnJPdFcrYzVheUVhdTJUbVNRCmorT0E5WnUzRWM5M0l5UzRiSWJINlFDaG1CMEkrYWk4VTk3SExhUjBVZUVHZWVpUVo5bktseUltT3lJcHV2dXcKUGV2ckdYZnFVRGhlVGNVQ0F3RUFBYU9DQVZRd2dnRlFNQWtHQTFVZEV3UUNNQUF3RVFZSllJWklBWWI0UWdFQgpCQVFEQWdaQU1ETUdDV0NHU0FHRytFSUJEUVFtRmlSUGNHVnVVMU5NSUVkbGJtVnlZWFJsWkNCVFpYSjJaWElnClEyVnlkR2xtYVdOaGRHVXdIUVlEVlIwT0JCWUVGS0l1RkkrVHBTWUhzaXQ5bUhxY3pMOGExMitnTUlHMkJnTlYKSFNNRWdhNHdnYXVBRlBDUytmMWNpWUlzV24vdEI4eEZZa0xjZ1N2SW9ZR09wSUdMTUlHSU1Rc3dDUVlEVlFRRwpFd0pRVERFVE1CRUdBMVVFQ0F3S1RXRnNiM0J2YkhOcllURVBNQTBHQTFVRUJ3d0dTM0poYTI5M01ROHdEUVlEClZRUUtEQVpHYjI4Z1EyOHhFakFRQmdOVkJBc01DVVp2YnlCRGJ5QklVVEVQTUEwR0ExVUVBd3dHUm05dklFTnYKTVIwd0d3WUpLb1pJaHZjTkFRa0JGZzV2Wm1acFkyVkFabTl2TG1OdmJZSUNFQUF3RGdZRFZSMFBBUUgvQkFRRApBZ1dnTUJNR0ExVWRKUVFNTUFvR0NDc0dBUVVGQndNQk1BMEdDU3FHU0liM0RRRUJDd1VBQTRJQkFRQ0toL2hlCk5IbEZQeXZXWlpYNDhlQ2Q1NEZGQ3NrNHpuUitvQlkxYVcxancrN0FGcUt4cUVmZWNkSlIrdUMrczBLVmNhUlEKWjZGcTUwaWNURUs3K1RmNHM0TGMvLytOSi92UEIxd1ZrNW1Rd3Zyb1RTcWZuOXRDdWNFL3gwa0FJSS80QmpZegpKUUNyZnlyVWlDbVBoME1vQTZ3Q0RiNlpMQ0M3TlFmVkFuN2RZNHVvWjNObDR3TW9IOGpSVWhYaHRKNjcwT3pGClpCc2lzSHBKS3RSaWJkOUNvY3I0dHdvTUFNbndpL3k0NWY4SC8yVm9FdnNKdzBDY00wK3FQUWVqSmNyMXh5TVAKMWZMQmNNTHlpeGNQNWYxLzlzWVRhQWN6TkU0eU1ReE9DS2hnZnBXcGE1Y2hVU2F6R2RnalBmNERaL0FSQSswWAowMmMrdGhMVStXMUdkbUhTCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0K"
```

Comment:
wow, that's huge. Could we add an issue to support reading these values from a file(s) and/or the environment variable(s)?

Reply:
we could - but I believe we'd _generally_ like to add support for reading environment variables as input for the pieces of the object model... but I don't see such an issue, adding it now: https://github.com/golemfactory/dapp-runner/issues/48

still, this is releated a bit to a general idea of having a Golem Application Object Model and some notation referencing objects and fields in the object model... maybe then the environment could be a special object in this model - or could be explicitly included as a type of "resource"

Reply:
anyway, the way I see it, there are more problems with these values and generally with the manifest signing model... 

and the size of those fields is the _least_ of them...